data/brands/shop/furniture.json

```
@@ -454,7 +469,7 @@
       },
       "tags": {
         "brand": "Habitat",
-        "brand:wikidata": "Q689059",
+        "brand:wikidata": "Q28232260",
         "brand:wikipedia": "fr:Habitat (entreprise)",
```

I guess this could link to the en version of the page (which is a sitelink for Q28232260).  
The en version explains that the UK and International organizations have separated..

```suggestion
        "brand:wikipedia": "en:Groupe Habitat",
```

---

Done thanks. I wasn't sure of the policy regards which language should be linked to for non-English websites.